src/formats/bbcode.js

```diff
@@ -2242,40 +2284,33 @@
 		};
 
 		/**
-		 * Populates tagsToBBCodes and stylesToBBCodes to enable faster lookups
+		 * Populates tagsToBBCodes and stylesToBBCodes for easier lookups
 		 *
 		 * @private
 		 */
 		function buildBbcodeCache() {
-			each(bbcodeHandlers, function (bbcode) {
-				var	isBlock,
+			each(bbcodeHandlers, function (bbcode, handler) {
+				var
 					tags   = bbcodeHandlers[bbcode].tags,
-					styles = bbcodeHandlers[bbcode].styles;
+					styles = bbcodeHandlers[bbcode].styles,
+					// TODO: Move this to utils
+					isObject = o => o instanceof Object
```

Comment:
For now the JS needs to remain compatible with IE and other old browsers so the editor can run in source mode when `runWithoutWysiwygSupport` is set.

I should have said that when I deprecated IE support., it's why I haven't updated the code to using the modern features yet, no matter how much I really want to.

This would either have to wait until the next breaking release or be converted to ES5 (urgh).